src/main/java/com/github/manolo8/darkbot/config/Config.java

```diff
@@ -118,6 +118,7 @@ public static class Roaming {
             public @Option boolean SEQUENTIAL = false;
             public @Option boolean ONLY_KILL_PREFERRED = false;
             public @Option boolean ENEMY_CBS_INVISIBLE = false;
+            public @Option boolean RADIATION = false;
```

Comment:
This has nothing to do with *roaming*, roaming is for the bot just randomly moving around the map, not for traveling with a specific destination.

Put this setting in the misc category alongside Avoid mines, as "AVOID_RADIATION", and rename it accoordingly (default to true)